ui/v2.5/src/locales/en-GB.json

```diff
@@ -405,7 +405,7 @@
         "auto_tagging_all_paths": "Auto Tagging all paths",
         "auto_tagging_paths": "Auto Tagging the following paths"
       },
-      "auto_tag_based_on_filenames": "Auto-tag content based on filenames.",
+      "auto_tag_based_on_filenames": "Auto-tag content based on filepaths and filenames.",
```

Comment:
```suggestion
      "auto_tag_based_on_filenames": "Auto-tag content based on file paths and names.",
```

Reply:
I'm not quite sure why you didn't go with my suggestion; file paths and names is redundant.